javascript/javascript1/week4/voiceAssistant.js

```diff
@@ -0,0 +1,183 @@
+let userName = '';
+const toDoList = [];
+
+
```

Comment:
Really good decision to extract logic of each conditional into different smaller functions - so code stayed easy to read and furthermore to maintain and debug
Good job!